### PR TITLE
adds potato chips to snacks

### PR DESCRIPTION
**Pull request by idalindegaard**
### This PR adds | fixes:
- Adds a delicious potato chip recipe to the meatballs repo

**How to prepare for test**:
- [ ] `ssh` to ...
- [ ] Install on stage:
`bash servers/resources/SERVER.scilifelab.se/update-[THIS_TOOL]-stage.sh [THIS-BRANCH-NAME]`

### How to test:
-

### Expected outcome:
- [ ]

### Review:
- [ ] Code approved by
- [ ] Tests executed by
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: docs/Snacks/potato_chips.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# **Potato chips**
+
+![potato chips](https://simple-veganista.com/wp-content/uploads/2012/07/best-homemade-baked-potato-chip-recipe-1-360x360.webp)
+
+*So simple, healthy and easy to make, once you make your own crispy homemade baked potato chips, there’s no going back to the bag!*
+
+## **Ingredients**
+
+- 2 medium potatoes
```
Comment (cesve) [inline docs/Snacks/potato_chips.md:9]: Perhaps specify which kind of potatoe to use?

**Review by cesve — APPROVED** [review]
Looks very good!!

File: docs/Snacks/potato_chips.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- 2 tsp olive oil
+- Himalayan salt, to taste – go easy as they can get too salty
+
+
+---
+**Prep Time:** 10 min  
+**Cooking Time:** 20 min
```
Comment (VasKyriak) [inline docs/Snacks/potato_chips.md:16]:
```suggestion
- Himalayan salt, to taste – go easy as they can get too salty


---
**Prep Time:** 15 min   
**Cooking Time:** 20 min  
```

File: docs/Snacks/potato_chips.md (hunk 1 above)
Comment (idalindegaard) [reply docs/Snacks/potato_chips.md:9]: I'd rather keep the options open

**Pull request merged**